composer.json

```diff
@@ -200,6 +201,8 @@
         "phpunit/phpunit": "<8.0",
         "roave/better-reflection": "<4.12.2 || >=6.0",
         "symfony/dependency-injection": "5.4.16 || 6.0.16 || 6.1.8 || 6.2.0 || 6.2.1",
+        "symfony/framework-bundle": "6.1.0 || 6.1.1",
+        "symfony/twig-bundle": "6.1.0",
```

Comment:
These conflicts are unnecessary. We can require `^6.1.2` and `^6.1.1` above instead.